sdk/python/express_relay/searcher/examples/simple_searcher_svm.py

```diff
@@ -119,7 +131,46 @@ async def get_mint_decimals(self, mint: Pubkey) -> int:
         return self.mint_decimals_cache[str(mint)]
 
     async def assess_opportunity(self, opp: OpportunitySvm) -> BidSvm | None:
```

Comment:
remove None from the return type if `assess_opportunity` can't return None anymore